src/integrationTest/java/org/opensearch/test/framework/cluster/TestRestClient.java

```diff
@@ -32,8 +32,6 @@
 import java.io.UnsupportedEncodingException;
 import java.net.InetAddress;
 import java.net.InetSocketAddress;
-import java.net.URI;
-import java.net.URISyntaxException;
```

Comment:
It appears to be complaining about the imports for some reason... I know this is a backport but would you be able to run spotlessApply to resolve whatever the complaint it has is?

Reply:
This is from how I handled the conflicts - just pushed a change to fix it